### ISSUE TITLE
Not working with auto_now_add and auto_now

### ISSUE THREAD

**Issue by Kazauwa** [yandex/zero-downtime-migrations#20]
Hi!
Your library solves our problem perfectly, but I found that it does not recognize `ALTER TABLE ADD COLUMN SET DEFAULT` migration when using `auto_now_add=True` and `auto_now=True` with DateTimeField:
```
migrations.AddField(
    model_name='mymodel',
    name='created_at',
    field=models.DateTimeField(auto_now_add=True, null=True),
)
```
Here's what `./manage.py sqlmigrate` says:
```
BEGIN;
--
-- Add field created_at to mymodel
--
ALTER TABLE "myapp_mymodel" ADD COLUMN "created_at" timestamp with time zone DEFAULT '2019-09-02T14:51:19.633657+00:00'::timestamptz NULL;
ALTER TABLE "myapp_mymodel" ALTER COLUMN "created_at" DROP DEFAULT;
COMMIT;
```
But if I replace `auto_now_add=True` with `default=localtime`, everything works like a charm

**Comment by Smosker** [comment]
Hello! Thanks for your report, will look into it.

**Comment by Smosker** [comment]
upload fix in version 0.8, please, try now with fresh version

**Comment by Kazauwa** [comment]
It works like a charm! Thank you!
